conf/test_glimpse2.config

```diff
@@ -34,6 +34,13 @@ params {
     tools  = "glimpse2"
 }
 
-withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:VCF_CHUNK_GLIMPSE:GLIMPSE_CHUNK' {
-    ext.args   = ["--window-size 10000", "--window-count 400", "--buffer-size 5000", "--buffer-count 30"].join(' ')
+process {
+    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:VCF_CHUNK_GLIMPSE:GLIMPSE_CHUNK' {
+        ext.args   = ["--window-size 10000", "--window-count 400", "--buffer-size 5000", "--buffer-count 30"].join(' ')
+    }
+
+    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:VCF_CHUNK_GLIMPSE:GLIMPSE2_CHUNK' {
+        ext.args   = ["--window-mb 0.01", "--window-cm 0.01", "--window-count 200", "--buffer-mb 0.005", "--buffer-cm 0.005", "--buffer-count 30"].join(' ')
+    }
```

Comment:
If this configuration is essential for the pipeline to work correctly, shouldn't this be the default in the process `GLIMPSE2_CHUNK`? Why include this for the test only? 

I wonder the case when a user wants to run the pipeline with their samples and it fails in this step. The solution would be to add these arguments as external parameters, but currently we don't allow that possibily. 

At this initial stage I don't think we should allow customization of such arguments via external params, as I think the default arguments will suffice. Could we consider moving this to the default config in `panel_prep.config` and in `imputation_glimps2.config`?

Reply:
These modifications are only necessary because we are not working on a full chromosomes and because our test region is really small.
The default options of glimpse and glimpse2 should be correct when working on bigger regions.
Moreover these parameters might create too much chunk if done on a full chromosome so I would only let them in the test and not add them to the default configurations.